__tests__/start.test.js

```diff
@@ -52,4 +52,54 @@ describe.only("start api", () => {
 
     global.config.confirmFirstAddress = null;
   });
+
+  it("should start a MultiSig wallet if multisig is true", async () => {
+    global.config.multisig = TestUtils.multisigData;
+    TestUtils.stopWallet()
+
+    let response1 = await TestUtils.request
+      .post("/start")
+      .send({ seedKey: TestUtils.seedKey, "wallet-id": TestUtils.walletId, "multisig": true });
+    expect(response1.status).toBe(200);
+    expect(response1.body.success).toBeTruthy();
+
+    let response2 = await TestUtils.request
+      .get("/wallet/address")
+      .query({ index: 0 })
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response2.status).toBe(200);
+    expect(response2.body.address).toBe(TestUtils.multisigAddresses[0]);
+
+    global.config.multisig = {};
+  });
+
+  it("should not start a incorrectly configured MultiSig if multisig is true", async () => {
+    global.config.multisig = {};
+
+    let response1 = await TestUtils.request
+      .post("/start")
+      .send({ seedKey: TestUtils.seedKey, "wallet-id": TestUtils.walletId, "multisig": true });
+    expect(response1.status).toBe(200);
+    expect(response1.body.success).toBeFalsy();
+
+    global.config.multisig = TestUtils.multisigData;
```

Comment:
I think we should improve the way we mock the configuration (using jest mock instead of global), this would solve the problem and wouldn't change the readability of the tests.
A change like this would be better on it's own PR, and since using the global works (at least for now) we can use it like this until we change the configuration mock.